target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -67,6 +67,10 @@ mediatek,linkit-smart-7688)
 oraybox,x1)
 	ucidef_set_led_netdev "wifi" "wifi" "blue:status" "wlan0"
 	;;
+qding,qc202)
+	ucidef_set_led_netdev "ethernet" "ethernet" "ethernet" "eth0"
+	ucidef_set_led_wlan "wifi" "wifi" "wifi" "phy0tpt"
```

Comment:
This can be done in dts now. Checkout linux,default-trigger property.

Reply:
Do I need to delete them in 02_led or do they exist in both 02_led and dts? I think LEDs don't need a default trigger in dts. This is a generic LED. I think it's better to keep it off if there is no configuration.

Reply:
Drop these. These are not the default function of the vendor firmware.

Reply:
OK